templates/ssr/package.json

```diff
@@ -5,7 +5,7 @@
 	"scripts": {
 		"dev": "vite",
 		"build": "vite build",
-		"preview": "vite build && vite preview"
+		"preview": "vite preview"
```

Comment:
Alternatively, could edit the base template to also be `build & preview`, I have no opinions on which. Just preferably they'd match.